docs/cluster/config.md

```diff
@@ -20,6 +20,9 @@ export CORTEX_BUCKET="cortex-[RANDOM_ID]"
 # The AWS region Cortex will use
 export CORTEX_REGION="us-west-2"
 
+# The AWS zones Cortex will use
```

Comment:
What is the format? comma separated? Can you mention the format and/or give an example?